Ember/app/templates/login.hbs

```diff
@@ -0,0 +1,44 @@
+<body id="login-body">
+    
+<div class="container login-container">
+    <div class="row">
+        <div class="col-md-4">
+            
```

Comment:
Ako je jedina svrha ove prazne 4 kolone da ove Slijedeće budu u sredini, to može postići i dodavanjem `offset-*` klase na ovaj ispod